selfdrive/car/__init__.py

```diff
@@ -301,3 +301,13 @@ def create_carinfo_map(cls) -> dict[str, CarInfos]:
   @classmethod
   def with_flags(cls, flags: IntFlag) -> set['Platforms']:
     return {p for p in cls if p.config.flags & flags}
+
+
+class CarFlags(IntFlag):
+  def check(self, flags: int) -> bool:
+    if self not in [f.value for f in self.__class__]:
```

Comment:
```suggestion
    if len(self) != 1:
```

Reply:
What is the difference between `SubaruFlags.GLOBAL_GEN2.any(SubaruFlags.GLOBAL_GEN2)` and `SubaruFlags.GLOBAL_GEN2.check(SubaruFlags.GLOBAL_GEN2)`?

Reply:
```python
In [5]: flags = SubaruFlags.GLOBAL_GEN2 #| SubaruFlags.HYBRID
   ...: (SubaruFlags.GLOBAL_GEN2 | SubaruFlags.HYBRID).all_in(flags)
Out[5]: False

In [6]: flags = SubaruFlags.GLOBAL_GEN2 | SubaruFlags.HYBRID
  ...: (SubaruFlags.GLOBAL_GEN2 | SubaruFlags.HYBRID).all_in(flags)
Out[6]: True
```

Reply:
We don't use is_set, do we need it?